cc/algorithms/count.h

```diff
@@ -129,6 +130,7 @@ class Count<T>::Builder
     ASSIGN_OR_RETURN(mechanism, AlgorithmBuilder::UpdateAndBuildMechanism());
 
     return absl::WrapUnique(new Count<T>(AlgorithmBuilder::GetEpsilon().value(),
+                                         AlgorithmBuilder::GetDelta().value_or(1e-5),
```

Comment:
I think the default value for delta should be 0.  This is still backwards compatible as the default mechanism is Laplace, which does not require any delta at all.

Reply:
Fixed in the next commit